src/Microsoft.Azure.SignalR/ServiceOptionsExtensions.cs

```diff
@@ -14,6 +15,10 @@ public static void Validate(this ServiceOptions options)
             {
                 throw new AzureSignalRInvalidServiceOptionsException("MaxPollIntervalInSeconds", "[1,300]");
             }
+            if (!string.IsNullOrEmpty(options.ApplicationName) && !Regex.IsMatch(options.ApplicationName, "^[a-zA-Z][a-zA-Z0-9_]*$"))
+            {
+                throw new AzureSignalRInvalidServiceOptionsException("ApplicationName", "prefixed with alphabetic characters and only contain alpha-numeric characters or underscore");
```

Comment:
`Application name should be prefixed with...`

Reply:
> Base message is formatted like "Property '{propertyName}' value should be {validScope}." https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRInvalidServiceOptionsException.cs#L13.